source/includes/usage-examples/SampleWriteApp.scala

```diff
@@ -0,0 +1,24 @@
+
+import org.mongodb.scala._
+import org.mongodb.scala.bson.Document
+import org.mongodb.scala.model._
+import org.mongodb.scala.model.Filters._
+import org.mongodb.scala.result._
+import org.mongodb.scala.model.Updates._
+
+object SampleWriteApp {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+    val database: MongoDatabase = mongoClient.getDatabase("<database name>")
+    val collection: MongoCollection[Document] = database.getCollection("<collection name>")
+
+    // Start example code here
+
+    // End example code here
+
+    // Wait for the operations to complete before closing client
```

Comment:
```suggestion
    // Wait for the operations to complete before closing client
    // Note: This example uses `Thread.sleep` for brevity 
    // this does not guarantee all operations will be completed in time.
```

I think its important to note to users the sleep provides no guarantees.